docs/tables/aws_iam_service_specific_credential.md

```diff
@@ -16,7 +16,7 @@ from
   aws_iam_service_specific_credential;
 ```
 
-### IAM user details for the service specific credential
+### IAM user details for the service specific credentials
```

Comment:
Can a user have multiple service specific credentials? If so, do we get multiple rows from this example?